apps/assisted_tagging/tasks.py

```diff
@@ -24,7 +26,9 @@ def _get_existing_tags_by_tagid():
             tag.tag_id: tag  # tag_id is from deepl
             for tag in AssistedTaggingModelPredictionTag.objects.all()
         }
-    response = requests.get(DeeplServiceEndpoint.ASSISTED_TAGGING_TAGS_ENDPOINT).json()
+
+    headers = {'Authorization': f'Token {settings.DEEPL_SERVER_TOKEN}'}
+    response = requests.get(DeeplServiceEndpoint.ASSISTED_TAGGING_TAGS_ENDPOINT, headers=headers).json()
```

Comment:
We have this already, let use that instead
```
from deepl_integration.handlers import BaseHandler as DeeplHandler
```

```suggestion
    response = requests.get(DeeplServiceEndpoint.ASSISTED_TAGGING_TAGS_ENDPOINT, headers=DeeplHandler.REQUEST_HEADERS).json()
```